docs/current-1.1/tools/dotnet-nuget-locals.md

```diff
@@ -0,0 +1,86 @@
+---
+title: dotnet-nuget-locals command | .NET Core SDK
+description: The dotnet-nuget-locals command clears or lists local NuGet resources such as http-request cache, temporary cache, or machine-wide global packages folder. 
+keywords: dotnet-nuget-locals, CLI, CLI command, .NET Core
+author: karann-msft
+ms.author: mairaw
+manager: wpickett
+ms.date: 11/10/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 8440229e-317e-4dc1-9463-cba5fdb12c3b
+---
+
+#dotnet-nuget-locals
+
+[!INCLUDE[preview-warning](../../includes/warning.md)] 
+
+## Name 
+dotnet-nuget-locals -- Clears or lists local NuGet resources such as http-request cache, temporary cache, or machine-wide global packages folder. 
+
+## Synopsis
+
+`dotnet nuget locals <cache_location> [--clear|--list] [--help] [--force-english-output]`
+
+where `<cache_location>` is one of the following values: `all`, `http-cache`, `packages-cache`, `global-packages`, or `temp`.
+
+## Description
+
+The `dotnet nuget locals` command clears or lists local NuGet resources such as http request cache, temporary cache, or machine-wide global packages folder.
+
+## Arguments
+
```

Comment:
consider putting the argument and its description on separate lines, the way you do for options.